examples/microwave-oven-app/microwave-oven-common/src/microwave-oven-device.cpp

```diff
@@ -61,6 +94,17 @@ Protocols::InteractionModel::Status ExampleMicrowaveOvenDevice::HandleModifyCook
     return Status::Success;
 }
 
+CHIP_ERROR ExampleMicrowaveOvenDevice::GetWattSettingByIndex(uint8_t index, uint16_t & wattSetting)
+{
+    // placeholder implementation
+    if (index > (ArraySize(mWattSettingList) - 1))
```

Comment:
I would avoid substraction on unsigned numbers just in case. This seems to be the same as `(index + 1) > ArraySize(mWattSettingList)` or `index >= ArraySize(mWattSettingList)`.

The latter is shorter and does not use substration. Please pick one of the 2 versions instead.

Reply:
You could also use macros:

```cpp
VerifyOrReturnError(index < ArraySize(mWattSettingList), CHIP_ERROR_NOT_FOUND);
```